src/olympia/git/models.py

```diff
@@ -0,0 +1,26 @@
+from django.db import models
+
+from olympia.addons.models import Addon
+from olympia.amo.models import ModelBase
+
+
+class AddonGitExtraction(ModelBase):
+    """
+    This is an add-on related model that stores information related to the git
+    extraction (for code-manager).
+    """
+
+    addon = models.OneToOneField(
+        Addon, on_delete=models.CASCADE, primary_key=True
```

Comment:
mmm, `primary_key`, are you sure?

Reply:
it was suggested here https://github.com/mozilla/addons-server/pull/13940#discussion_r406134574